apparmor.d/groups/kde/mariadbd_akonadi

```diff
@@ -0,0 +1,54 @@
+abi <abi/3.0>,
+
+#include <tunables/global>
+
+@{exec_path} = @{bin}/mariadb{,-*} @{bin}/mariadbd @{bin}/mysql @{bin}/mysqld{,*}
```

Comment:
That won't work. Doing so you are going to confine a mariadbd server. Not the database of akonadi.

Reply:
I'm not sure that I understand. In my understanding AppArmor profiles do not confine data(bases) (unless specific data is protected by including,e.g., the private-files or sensitive-home abstractions or adding specific deny rules) but rather applications are confined by allowing them to do/access only what's necessary. So how is this profile (and the original one) different from all the other ones? 

FWIW, I've been using this profile for quite some time in enforce mode:
`/usr/bin/mariadbd (8323) mariadbd_akonadi`

I'll remove the bash abs and replace the nameservice abs with nameservice-strict  and see what's necessary to add.